var/spack/repos/builtin/packages/gdal/package.py

```diff
@@ -414,8 +414,11 @@ def configure_args(self):
             args.append('--with-curl=no')
 
         if '+xml2' in spec:
-            args.append('--with-xml2={0}'.format(
-                join_path(spec['libxml2'].prefix.bin, 'xml2-config')))
+            if spec.satisfies('@:2.4.4'):
```

Comment:
```suggestion
            if spec.satisfies('@:2'):
```
In case they add a new patch version